client/src/management/commands.py

```
@@ -0,0 +1,167 @@
+        # Prepare gas kwargs
+        gas_kwargs = {}
+        if gas_limit:
+            gas_kwargs["gas_limit"] = gas_limit
+        if gas_multiplier != 1.1:
+            gas_kwargs["gas_multiplier"] = gas_multiplier
+
+        # Submit rewards
+        avs_owner.submit_rewards_for_interval(interval, **gas_kwargs)
+
+        logger.info(f"🎉 Successfully submitted rewards for interval {interval}")
+        typer.echo(f"✅ Rewards submitted successfully!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)